CHANGELOG.md

```diff
@@ -112,6 +115,9 @@ replace (
 ### Bug Fixes
 
 * (x/mint) [#12384](https://github.com/cosmos/cosmos-sdk/pull/12384) Ensure `GoalBonded` must be positive when performing `x/mint` parameter validation.
+* (simapp) [#12437](https://github.com/cosmos/cosmos-sdk/pull/12437) fix the non-determinstic behavior in simulations caused by `GenTx` and check
```

Comment:
We removed that in our security fix, however this should not be removed, as it is obviously still included in these versions.